shub/image/build.py

```diff
@@ -70,6 +75,26 @@ def build_cmd(target, version, skip_tests):
         test_mod.test_cmd(target, version)
 
 
+def _create_or_update_progressbar(bar, event, verbose):
+    """Helper to create progress bar to track progress."""
+    step_row = STEP_REGEX.match(event['stream'])
```

Comment:
sometimes there are substeps, caused by build triggers, do you ignore them properly?

<img width="704" alt="1__alex_mbp____dev_workspace_scrapinghub_proj_scrapy-first-project_hackershelf__zsh__and_progress_bar_for_shub_image_build_operation_by_vshlapakov_ _pull_request__251_ _scrapinghub_shub" src="https://cloud.githubusercontent.com/assets/744331/24721981/ec22fb36-1a39-11e7-8e53-01124384a6b6.png">